netbox_netprod_importer/vendors/juniper/base.py

```diff
@@ -171,3 +171,22 @@ def _gen_rpc_lldp_neighbours(self):
         xml_tree = get_lldp_neighbours_el.getroottree()
 
         return lxml.etree.tostring(xml_tree).decode()
+
```

Comment:
I would raise a `NotImplementedError` for these functions

Reply:
Redid it.
I do not like to cause exceptions .....